probe-rs/src/debug/debug_info.rs

```diff
@@ -1129,6 +1120,29 @@ impl DebugInfo {
 
         Some((file_name, directory))
     }
+
+    // Return the compilation unit that contains the given address
+    pub(crate) fn compile_unit_info(
+        &self,
+        address: u64,
+    ) -> Result<&super::unit_info::UnitInfo, DebugError> {
+        for header in &self.unit_infos {
+            match self.dwarf.unit_ranges(&header.unit) {
+                Ok(mut ranges) => {
+                    while let Ok(Some(range)) = ranges.next() {
+                        if (range.begin <= address) && (range.end > address) {
```

Comment:
Don't we have `range.contains` for this? I remember adding that for some Range type a few weeks back.

Reply:
This is a `gimli::Range` ... does not have contains()  ;-(

Reply:
https://github.com/probe-rs/probe-rs/blob/98e21bc17b754a6ef2787a7819bc2f2096dbfdab/probe-rs/src/debug/unit_info.rs#L2236

Reply:
Done. Thanks for the pointer:)